test/achievement-specs.js

```diff
@@ -74,25 +114,74 @@ describe('achievement', () => {
   })
 
   context('POST /achievement', () => {
-    it('should create achievement with sequence=sequence, name=name, and distance=distance', async () => {
+    it('should create achievement with sequence=sequence, name=name, distance=distance,' +
+      'description=description, icon_name=icon_name, map_image=map_image,' +
+      'title=title, subtitle=subtitle, content=subtitle, and media=media', async () => {
       let sequence = 1
       let name = 'London'
       let distance = 100
+      let description = ''
+      let iconName = 'pakistan'
+      let mapImage = 'test.png'
+      let title = 'title'
+      let subtitle = 'subtitle'
+      let content = 'content with html tags'
+      let media = 'photo:photo_1.png video:video_1.png'
       await koaRequest
         .post('/achievement')
-        .send({sequence, name, distance})
+        .send({
+          sequence,
+          name,
+          distance,
+          description,
+          icon_name: iconName,
+          map_image: mapImage,
+          title,
+          subtitle,
+          content,
+          media
+        })
         .expect(201)
         .then(response => {
           response.body.sequence.should.equal(sequence)
           response.body.name.should.equal(name)
           response.body.distance.should.equal(distance)
+          response.body.description.should.equal(description)
+          response.body.icon_name.should.equal(iconName)
+          response.body.map_image.should.equal(mapImage)
+          response.body.title.should.equal(title)
+          response.body.subtitle.should.equal(subtitle)
+          response.body.content.should.equal(content)
+          response.body.media.should.equal(media)
         })
     })
     it('should return 409 if achievement name conflict', async () => {
-      let a2 = await models.db.achievement.create({sequence: 1, name: 'Paris', distance: 100})
+      let a2 = await models.db.achievement.create({
+        sequence: 1,
+        name: 'London',
+        distance: 100,
+        description: 'AKF UK',
+        icon_name: 'greatbritain',
+        map_image: 'challenge1_0.png',
+        title: 'Article1',
+        subtitle: 'Author1',
+        content: 'content1 with html tags',
+        media: 'photo:photo1_url video:video1_url'
+      })
       await koaRequest
         .post('/achievement')
-        .send({sequence: 0, name: a2.name, distance: a2.distance})
+        .send({
+          sequence: 0,
+          name: a2.name,
+          distance: a2.distance,
+          description: a2.description,
+          icon_name: a2.flag_ame,
```

Comment:
What is this property? Not sure that this is valid (likely undefined)
```suggestion
          icon_name: a2.icon_name,
```